src/components/ComputeModal.js

```diff
@@ -934,7 +934,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
         p([
           'You are about to create a virtual machine using an unverified Docker image. ',
           'Please make sure that it was created by you or someone you trust, using one of our ',
-          h(Link, { href: terraBaseImages, ...Utils.newTabLinkProps }, ['base images.']),
+          h(Link, { 'aria-label': 'Terra base images', href: terraBaseImages, ...Utils.newTabLinkProps }, ['base images.']),
```

Comment:
It seems like we should just normalize this text:
```suggestion
          h(Link, { href: terraBaseImages, ...Utils.newTabLinkProps }, ['Terra base images.']),
```